README.md

```diff
@@ -82,7 +82,7 @@ It is a calculater has been created by React
 
```

Comment:
- [ ] Kindly remove these marked boilerplate files as they would not be used and so we should keep the code base clean.

  public folder | src folder
  :---:|:---:
  ![iduw](https://user-images.githubusercontent.com/36542986/234883387-2a03a900-a514-42da-9d77-aaaa389caf3f.jpg) | ![ifofw](https://user-images.githubusercontent.com/36542986/234883372-97cf7859-daa3-4a4e-a035-a5afa18c7a06.jpg)